KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyView.swift

```diff
@@ -19,38 +20,43 @@ struct KHLoyaltyViewID {
     public static let burnPointsSwitch = "burn_points_switch"
     public static let infoView = "info_view"
     public static let infoLabel = "info_label"
+    public static let balanceView = "balance_view"
+    public static let balanceLabel = "balance_label"
 }
 
-final class KarhooLoyaltyView: UIStackView {
-    
-    private let drawAnimationTime: Double = 0.45
-    private let standardSpacing: CGFloat = 12.0
-    private let smallSpacing: CGFloat = 4.0
-    private let mediumCornerRadius: CGFloat = 8.0
-    private let cornerRadius: CGFloat = 3.0
-    private let borderWidth: CGFloat = 1.0
+final class KarhooLoyaltyView: UIView {
     
     private var presenter: LoyaltyPresenter?
     private var didSetupConstraints: Bool = false
     private var topSwitchConstraint: NSLayoutConstraint?
     private var bottomSwitchConstraint: NSLayoutConstraint?
     
     // MARK: - UI
+    private lazy var containerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.accessibilityIdentifier = KHLoyaltyViewID.containerStackView
+        stackView.spacing = UIConstants.Spacing.medium
+        stackView.axis = .vertical
```

Comment:
This is the default value. Unless you want to empathize this property, this is not needed.